tests/unit/event_callback.c

```diff
@@ -81,6 +81,16 @@ extern drizzle_return_t drizzle_event_callback(drizzle_st *con, short events,
   return DRIZZLE_RETURN_OK;
 }
 
+extern void drizzle_free_context(drizzle_st *con, void *context);
+extern void drizzle_free_context(drizzle_st *con, void *context)
+{
+  (void)con;
```

Comment:
Why do you specify the name then ?

Reply:
Otherwise it results in an `unused variable warning`

Reply:
AFAIK compilers won't issue this warning if you don't specify a name:
```
extern void drizzle_free_context(drizzle_st *, void *context) 
```

Reply:
That results in : 

`../tests/unit/event_callback.c:85:1: error: parameter name omitted`

Reply:
compiler warning flags are on viagra

Reply:
That's why I don't want to do C++ anymore